MultiServer.py

```diff
@@ -811,6 +834,8 @@ def send_items_to(ctx: Context, team: int, target_slot: int, *items: NetworkItem
 
 def register_location_checks(ctx: Context, team: int, slot: int, locations: typing.Iterable[int],
                              count_activity: bool = True):
+    if ctx.hint_start_time == 0:
+        ctx.hint_start_time = time.time()
```

Comment:
time.time has platform dependant quirks; it might be better to use datetime.utcnow()